### PR TITLE
Add KeyShot 8.2.80

### PR DESCRIPTION
**Pull request by zthxxx**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by zthxxx** [comment]
@vitorgalvao 

all CI checks have passed, need merge plz

File: Casks/keyshot.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  version '8.2.80'
+  sha256 'c8df6bca2c0c89295ebcd109dd7b1a6b8ec5f58907b7ee6120a77b3b06d4bef9'
+
+  url "https://download.keyshot.com/keyshot8/keyshot_mac64_#{version}.pkg"
+  name 'KeyShot'
+  homepage 'https://www.keyshot.com/'
+
+  pkg "keyshot_mac64_#{version}.pkg"
+
+  uninstall pkgutil: [
```
Comment (vitorgalvao) [inline Casks/keyshot.rb]: String, not array.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Please be patient and don’t ping maintainers directly unless it’s an ongoing conversation. Six hours is definitely too soon to do it, we deserve a full night of sleep like anyone else.

**Comment by vitorgalvao** [comment]
Please don’t force push, always add commits, so we maintainers can easily see what changed.

**Comment by vitorgalvao** [comment]
Please don’t force push, always add commits so maintainers can easily see what changed.

**Comment by zthxxx** [comment]
> Please don’t force push, always add commits so maintainers can easily see what changed.

sorry. i have squashed my commits as a single file.

but i don't know how to recover this merge without force-pushed again

**Comment by vitorgalvao** [comment]
> but i don't know how to recover this merge without force-pushed again

Don’t worry about it, that’s a note for next time.

**Comment by zthxxx** [comment]
> Don’t worry about it, that’s a note for next time.

thanks, i keep in mind

**Pull request merged**